.gitignore

```diff
@@ -87,6 +87,12 @@ celerybeat.pid
 celerybeat-schedule
 
 # Deploy
-conf/Dockerrun.aws.json
+*Dockerrun.aws.json
 conf/nginx.ureport.conf
 conf/deploy.zip
+*.elasticbeanstalk
+
+# Elastic Beanstalk Files
+.elasticbeanstalk/*
+!.elasticbeanstalk/*.cfg.yml
+!.elasticbeanstalk/*.global.yml
```

Comment:
Yeah so `config.global.yml` is the correct place for

    deploy:
      artifact: "conf/deploy.zip"